src/client/clientlauncher.cpp

```diff
@@ -275,6 +275,10 @@ bool ClientLauncher::run(GameStartData &start_data, const Settings &cmd_args)
 			error_message = gettext("Connection error (timed out?)");
 			errorstream << error_message << std::endl;
 		}
+		catch (ShaderException &e) {
+			error_message = debug_describe_exc(e);
```

Comment:
`debug_describe_exc` puts quotes around the error message, which is unnecessary here. possible solutions:

- modify `debug_describe_exc` to not put quotes around the error message
- use `e.what()` instead of `debug_describe_exc` in this case

Reply:
The latter is better. `debug_describe_exc` doesn't serve its purpose here since we know which exact exception we will get and that the error is already user-friendly.

Reply:
I agree.